README.md

```diff
@@ -87,3 +87,7 @@ You can contribute to the project on github. Here's how you can do it.
 <a href="https://www.w3.org/html/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/html-5.png"/> </a> <a href="https://www.w3schools.com/css/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/css3.png"/> </a><a href="https://www.php.net/" target="_blank"> <img src="https://img.icons8.com/dusk/64/000000/php-logo.png"/>
 
 </br>
+<hr>
+## 🔖 Figma Link for this project
+<a href="https://www.figma.com/file/dz4JS3XBiGzTDGGIQ3PczB/Diagon-Alley?node-id=20%3A782" target="_blank"> Figma Designs </a>
+</br>
```

Comment:
export the Home page from the Figma file and attached it in here

Reply:
[here is the figma link](https://www.figma.com/file/dz4JS3XBiGzTDGGIQ3PczB/Diagon-Alley?node-id=20%3A782)

Reply:
Grant access

Reply:
no need to add .png to the repository, actually I said you to attach that image to the readme